consoles/sshVirtsh.pm

```diff
@@ -183,6 +206,17 @@ sub add_pty {
     $elem->setAttribute(port => $args->{port});
     $console->appendChild($elem);
 
+    if (!($self->vmm_family eq 'xen' && $self->vmm_type eq 'linux')) {
+        my $serial = $doc->createElement('serial');
+        $serial->setAttribute(type => 'pty');
+        $devices->appendChild($serial);
+
+        $elem = $doc->createElement('target');
+        $elem->setAttribute(type => 'isa-serial');
+        $elem->setAttribute(port => $args->{port});
```

Comment:
I guess it's really safer to do this intellegence from a bootloader_* test instead of here through magic variables.